.gitignore

```diff
@@ -16,6 +16,7 @@ lib-cov
 
 # Coverage directory used by tools like istanbul
 coverage
```

Comment:
Also remove this

Reply:
Are you sure? 

Every time jest is ran it's going to create a coverage report folder which isn't really ideal to expose in source control imo.

Reply:
Oh I mean we now have two "coverage" entry in gitignore, this one can be removed since you added a /coverage one

Reply:
Oh I mean we now have two "coverage" entry in gitignore, this one can be removed since you added a /coverage one